dds_cli/__main__.py

```diff
@@ -1216,6 +1216,35 @@ def delete_project(click_ctx, project: str):
             sys.exit(1)
 
 
+# -- dds project status busy -- #
+@project_status.command(name="busy", no_args_is_help=False)
+# Flags
+@click.option("--show", required=False, show_default=True, is_flag=True, help="Show busy projects")
+@click.pass_obj
+def get_busy_projects(click_ctx, show):
+    """Display True or False depending on if there are any busy projects.
```

Comment:
Update the docstring

Reply:
"Display True or False" should be changed to e.g. that it returns the number of busy projects.

Reply:
OMG, I was wondering what you mean ...